py3status/modules/mpd_status.py

```diff
@@ -91,24 +101,48 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 2
     format = '{state} [[[{artist}] - {title}]|[{file}]]'
-    hide_when_paused = False
-    hide_when_stopped = True
+    hide_when_pause = False
+    hide_when_play = False
+    hide_when_stop = True
     host = 'localhost'
     max_width = 120
+    mpd_no_auth = 'mpd: authentication failed'
+    mpd_no_conn = 'mpd: connection refused'
```

Comment:
can we use `state_no_connection` etc, that seems a better fit.  Or else just remove these for now and leave hard coded